src/main.ts

```diff
@@ -47,7 +47,7 @@ async function run(): Promise<void> {
 
     core.exportVariable('FULL_SHA', fullSha)
     core.exportVariable('SHORT_SHA', shortSha)
-    core.exportVariable('BRANCH_NAME', branchName)
+    core.exportVariable('BRANCH_NAME', branchName?.replace(/\//g, '-'))
```

Comment:
For the auto dev environment, we went further and replaced every character that isn't a-zA-Z0-9 with -

Reply:
This ended up being a quick fix but we should probably do the same here too.